dips/dip-3.md

```diff
@@ -42,7 +42,7 @@ Note that, after DPN launches, DN could use genesis transactions as a last resor
 
 The decision flow chart below summarizes the building blocks and how they will be used.
 
-![Decision flow](../static/img/DIP-3-decision-flow.png)
+<!-- ![Decision flow](../static/img/DIP-3-decision-flow.png) -->
```

Comment:
Wanted to point out that I had to disable this image. The new webpack config for upgraded docusaurus is more strict and throws an error on this image since it is missing. It is currently missing on prod as well
https://dip.diem.com/dip-3/

Reply:
Can't we just add the image to this PR?

Reply:
Definitely if you have it. Otherwise when you do have it later on you can send it to me and I can put it up in a future PR. This line won't change the view of this page since the image already did not exist

Reply:
![DIP-3-decision-flow](https://user-images.githubusercontent.com/3757713/104493176-8ed2eb00-5589-11eb-9e70-69cca1c6818e.png)

Here you go!